Content.Server/Body/Systems/RespiratorSystem.cs

```diff
@@ -421,3 +444,9 @@ public record struct InhaledGasEvent(GasMixture Gas, bool Handled = false, bool
 
 [ByRefEvent]
 public record struct ExhaledGasEvent(GasMixture Gas, bool Handled = false);
+
+[ByRefEvent]
+public record struct SuffocationEvent;
+
+[ByRefEvent]
+public record struct StopSuffocatingEvent;
```

Comment:
comments on these events at least detailing when/what they get raised on.

really all the events here should have this.